js/ui/overlay/ui.overlay.js

```diff
@@ -1506,6 +1421,9 @@ const Overlay = Widget.inherit({
                 this._initContainer(value);
                 this._invalidate();
                 this._toggleSafariScrolling();
+                if(this.option('dragEnabled') && !this.option('dragArea')) {
```

Comment:
`dragArea` can be specified but it can not contain `container`

Reply:
fixed